src/cguittfont/CGUITTFont.cpp

```diff
@@ -645,7 +658,7 @@ void CGUITTFont::draw(const core::stringw& text, const core::rect<s32>& position
 
 		CGUITTGlyphPage* page = n->getValue();
 
-		if (!use_transparency) color.color |= 0xff000000;
+		//if (!use_transparency) color.color |= 0xff000000;
```

Comment:
I'd just remove this instead of commenting it out, since the logic is now in the loop below